charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,187 @@
+    # limits:
+    #   cpu: 100m
+    #   memory: 60Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 20Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  logPath:
+  # The operator currently assumes a Docker container runtime path for the logs as the default, for other container runtimes you can set the location explicitly below.
+  # crio: /var/log
+    containerd: /var/log
+
+fluentbit:
+  image:
+    repository: "kubesphere/fluent-bit"
+    tag: "v1.9.9"
+  # fluentbit resources. If you do want to specify resources, adjust them as necessary
+  # You can adjust it based on the log volume.
+  resources: {}
+    # limits:
+    #   cpu: 500m
+    #   memory: 200Mi
+    # requests:
+    #   cpu: 10m
+    #   memory: 25Mi
+  # Specify custom annotations to be added to each FluentBit pod.
+  annotations: {}
+  # fluentbit.io/exclude: "true"
+  # prometheus.io/scrape: "true"
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  secrets: []
+  # List of volumes that can be mounted by containers belonging to the pod.
+  additionalVolumes: []
+  # Pod volumes to mount into the container's filesystem.
+  additionalVolumesMounts: []
+
+  # Remove the above empty volumes and volumesMounts, and then set volumes and volumesMounts as below if you want to collect node exporter metrics
+# volumes:
+#   - name: hostProc
+#     hostPath:
+#       path: /proc/
+#   - name: hostSys
+#     hostPath:
+#       path: /sys/
+# volumesMounts:
+#   - mountPath: /host/sys
+#     mountPropagation: HostToContainer
+#     name: hostSys
+#     readOnly: true
+#   - mountPath: /host/proc
+#     mountPropagation: HostToContainer
+#     name: hostProc
+#     readOnly: true
+
+  # Set a limit of memory that Tail plugin can use when appending data to the Engine.
+  # If the limit is reach, it will be paused; when the data is flushed it resumes.
+  # You can find more details here: https://docs.fluentbit.io/manual/pipeline/inputs/tail#config
+  # if the inbound traffic is less than 2.4Mbps, setting memBufLimit to 5MB is enough
+  # if the inbound traffic is less than 4.0Mbps, setting memBufLimit to 10MB is enough
+  # if the inbound traffic is less than 13.64Mbps, setting memBufLimit to 50MB is enough
+  input:
+    tail:
+      memBufLimit: 5MB
+    # uncomment below nodeExporterMetrics section if you want to collect node exporter metrics
+#   nodeExporterMetrics:
+#     tag: node_metrics
+#     scrapeInterval: 15s
+#     path:
+#       procfs: /host/proc
+#       sysfs: /host/sys
+
+  # Configure the output plugin parameter in FluentBit.
+  # You can set enable to true to output logs to the specified location.
+  output:
```

Comment:
So, what do we get by default? If the user just deploys it with no changes to the values, what do we get?
I think we should have a comment at least on that, or expose the default values explicitly here.

Reply:
Because the output to which plugins need to be configured by the user, for example, the output address of `elasticsearch`, where there is no default output location, needs to be filled in the `value.yaml` file by the user.

Reply:
> So, what do we get by default? If the user just deploys it with no changes to the values, what do we get? I think we should have a comment at least on that, or expose the default values explicitly here.

I don't think we should comment out all the `output` and `filter` section, there're already switches to enable/disable a specific output, no need to comment them out